robot/Cumulus/resources/DataImportPageObject.py

```diff
@@ -1,6 +1,6 @@
 from cumulusci.robotframework.pageobjects import ListingPage
 from cumulusci.robotframework.pageobjects import pageobject
-from locators_46 import npsp_lex_locators
+from locators_47 import npsp_lex_locators
```

Comment:
Here and in some other places below, I think this should be `from NPSP import npsp_lex_locators` rather than referring to a specific version. The idea is that the NPSP module has a `npsp_lex_locators` dict which it takes care of updating to hold the correct set of locators when the NPSP library is initialized. Then other modules can import the dict from NPSP without needing to know what version it is.

Reply:
made suggested change